packages/next/src/build/babel/plugins/next-page-config.ts

```diff
@@ -5,7 +5,7 @@ import type {
   Visitor,
   NodePath,
 } from 'next/dist/compiled/babel/core'
-import type { PageConfig } from 'next/types'
```

Comment:
Is this a public api that we are going to break? I dont know if `next/types` is something the user would expect to import from.

Reply:
Should be the same content. `next/types` used to resolve to `next/types/index.d.ts`. Now it resolves to `next/types.d.ts` which just reexports `next/dist/types.d.ts`. And since I just moved everything from `next/types/index.d.ts` into `next/src/types.ts`, the content should also be the same.

It's really just reorganizing files so that the program using the root tsconfig doesn't include Next.js source files.